frontend/components/puzzle4.jsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
+import styled from 'styled-components';
 
 import { Narration } from '../wrappers';
 import { isSolved } from '../helpers';
 import { KeypadContainer, Button, SubmitButton, ClearButton } from './buttonContants';
 import { getCurrentStage } from '../helpers';
 
+const SafeImage = styled.img.attrs(props => ({
+  src: props.src,
```

Comment:
dont need this if the prop is just being passed through to the underlying component

Reply:
true statement.